keras_retinanet/models/retinanet.py

```diff
@@ -25,14 +25,14 @@
 A dictionary mapping custom layer names to the correct classes.
 """
 custom_objects = {
-    'UpsampleLike'          : layers.UpsampleLike,
-    'PriorProbability'      : initializers.PriorProbability,
-    'RegressBoxes'          : layers.RegressBoxes,
-    'NonMaximumSuppression' : layers.NonMaximumSuppression,
-    'Anchors'               : layers.Anchors,
-    'ClipBoxes'             : layers.ClipBoxes,
-    '_smooth_l1'            : losses.smooth_l1(),
-    '_focal'                : losses.focal(),
+    'UpsampleLike'     : layers.UpsampleLike,
+    'PriorProbability' : initializers.PriorProbability,
+    'RegressBoxes'     : layers.RegressBoxes,
+    'FilterDetections' : layers.FilterDetections,
```

Comment:
Also for a future PR, but I think we should prefix out custom objects with `keras_retinanet.`, or maybe even use the full module path and class name. Otherwise it becomes hard (if not impossible) to mix retinanet with other models that use the same names in the same graph.